docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/15900_article.content.yml

```diff
@@ -256,3 +256,148 @@
     value: '2019-06-19'
   field_date_display_mode:
     value: 'reviewed'
+
+# Article Test Page for Pull-Quotes
```

Comment:
Can you please move this to its own file that is nicely named? This thing is a legacy file that used to be horribly named "article.content.yml". 

Move it to a new file named `10550_embed_pull_quote_test.content.yml`. That is where the rest of the embed tests live.